tool/tsh/db.go

```diff
@@ -640,12 +632,13 @@ type localProxyConfig struct {
 
 // prepareLocalProxyOptions created localProxyOpts needed to create local proxy from localProxyConfig.
 func prepareLocalProxyOptions(arg *localProxyConfig) (localProxyOpts, error) {
-	// If user requested no client auth, open an authenticated tunnel using
-	// client cert/key of the database.
 	certFile := arg.cliConf.LocalProxyCertFile
 	keyFile := arg.cliConf.LocalProxyKeyFile
```

Comment:
not sure what's the history on these two flags (`--cert-file`, `--key-file`). do we still need them?

Reply:
I don't know what the history of them is. If they provide cert and key files, the local proxy tunnel will use them until they expire, at which point local proxy middleware will try to renew certs in memory